.github/workflows/announce-releases.yml

```diff
@@ -0,0 +1,18 @@
+name: "Announce Release on Discord"
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  announce-on-discord:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send Discord notification
+        if: ${{ github.event_name == 'release' }}
```

Comment:
Are you sure this is necessary? Asking because here we use the same trigger, and I didn;t find the reason for this check: https://github.com/platformatic/platformatic/blob/main/.github/workflows/docker-release.yml

Reply:
I added that because I thought I could trigger also with `workflow_dispatch` event. I will remove such event and the check